features/FEATURE_BLE/source/gap/Gap.tpp

```diff
@@ -0,0 +1,866 @@
+/* mbed Microcontroller Library
```

Comment:
Ooohhh. This was deleted, renamed, then added.

I was wondering why it looked like the GH interface was being weird.

`git mv <file>{cpp,tpp}` appears to not have been done.